IBM/operand-deployment-lifecycle-manager
# scaffolded OperatorConfig API

opening PR now for easier reviewing since a lot of files are generated when a new API is added.

How to test:

Should be no change to functionality, just able to install it and see OperatorConfig CRD
